harvester/pmh_interface.py

```diff
@@ -34,6 +38,25 @@ def fetch_records(self, from_timestamp=None):
                 }
             )
             for metadata_record in metadata_records:
-                yield metadata_record
+                yield MSRecordParser(deepcopy(metadata_record.xml))
```

Comment:
Was this returning someting unintentionally mutable before? Would the internal state of sickle then potentially be changed and break things?

Reply:
It was known and handled earlier too: we had `lxml_record = etree.fromstring(etree.tostring(metadata_content.xml))` in the CLI to handle that, but this felt like a cleaner solution.